docs/getting-started.md

```diff
@@ -1,54 +1,49 @@
 > NOTE: We have yet to publish our initial release to npm. However, once we do, the installation
 instructions will work.
 
-# Getting Started
+# Getting started
 
 This guide will help you get started using MDLv2 on your own sites and within your own projects.
 
 > If you are interested in integrating MDLv2 into a framework, or building a component library for
 your framework that wraps MDLv2, check out our [framework integration guide](./integrating-into-frameworks.md).
 
-## MDL Quick Start: Building a simple greeting app
+## MDL quick start: building a simple greeting app
 
 The best way to learn any new technology is to get your hands dirty and build something with it, so
-that's what we'll do here! We'll be building a simple web page which lets you enter a first and/or
-last name, and then greets you using your name. You can see a finished example [here](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
+that is what we will do here!  We will be building a simple greeting page which lets you enter a name and greets you as such. Here is the [finished example](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
```

Comment:
We > you

Second person (mentioned in another part of the review as well further down.)